concordium-node/src/p2p/peers.rs

```diff
@@ -145,3 +154,47 @@ pub fn check_peers(node: &Arc<P2PNode>, peer_stats: &[PeerStats], attempted_boot
         }
     }
 }
+
+/// Calculate the `checked` average of the received and send bytes.
+/// If any of the arguments would imply in a non-defined or otherwise
+/// non-meaningful result, then the function returns (0, 0).
+fn calculate_average_throughput(
```

Comment:
I think the "correct" thing to do would be to return a result or option. 

These defaulting values are not in the spirit of the rest of the project.